tests/test_builders/test_build_latex.py

```diff
@@ -2019,9 +2019,11 @@ def test_latex_labels(app: SphinxTestApp) -> None:
         r'\label{\detokenize{otherdoc::doc}}'
     ) in result
 
-    # Embedded standalone hyperlink reference
+    # Named hyperlink reference with embedded alias reference
     # See: https://github.com/sphinx-doc/sphinx/issues/5948
     assert result.count(r'\label{\detokenize{index:section1}}') == 1
+    # https://github.com/sphinx-doc/sphinx/issues/13609
+    assert r'\phantomsection\label{\detokenize{index:id' not in result
```

Comment:
All other LaTeX labels (apart from `\phantomsection\label{\detokenize{index::doc}}` one sees at start of document body do not use `\phantomsection` because they follow either a `\section/\chapter` (et al.) or a `\caption` and Sphinx LaTeX writer then does not inject unneeded `\phantomsection`.  The named hyperlink reference with embedded aliases on the other hand caused with Docutils HEAD at r10151 or later an extra `\phantomsection\label` which this detects.